spec/characterization_services/mediainfo_characterization_service_spec.rb

```diff
@@ -109,7 +109,7 @@
       allow(video_track_attributes).to receive(:encoded_date).and_return Time.zone.parse("UTC 2010-02-12 13:45:09")
       allow(video_track_attributes).to receive(:producer).and_return("Test Video Producer")
       allow(video_track_attributes).to receive(:originalsourceform).and_return("DAV")
-      allow(video_track_attributes).to receive(:duration).and_return(0.984)
+      allow(video_track_attributes).to receive(:duration).and_return(984)
```

Comment:
Where can we document the units we expect for duration? maybe https://github.com/pulibrary/figgy/blob/b62011b8d64efa4ae08a3d149b08e8fe16adf7ff/app/nested_resources/file_metadata.rb#L26